app/components/works/embargo_component.html.erb

```diff
@@ -11,26 +11,30 @@
      </div>
    </div>
 
-   <div class="mb-5 row">
+   <div class="embargo-date" data-controller="embargo-date" data-action="error->embargo-date#error" data-target="edit-deposit.embargo-dateField">
+   <div class="mb-5 row" data-target="embargo-date.container">
      <div class="col-sm-10 release-option">
-       <%= form.radio_button :release, 'embargo', class: 'form-check-input' %>
-       <%= form.label :release_embargo, 'On this date' %>
+          <%= form.radio_button :release, 'embargo', class: 'form-check-input' %>
+          <%= form.label :release_embargo, 'On this date' %>
 
-       <div class="year">
-         <label for="work_embargo_year" class="visually-hidden">Embargo year</label>
-         <%= select_year Time.zone.today, { prefix: 'work', field_name: 'embargo_date(1i)', start_year: Time.zone.today.year, end_year: 3.years.from_now.year }, id: 'work_embargo_year', class: "form-control" %>
-       </div>
-       <div class="month">
-         <label for="work_embargo_month" class="visually-hidden">Embargo month</label>
-         <%= select_month nil, { prefix: 'work', field_name: 'embargo_date(2i)', prompt: 'month'}, id: 'work_embargo_month', class: "form-control" %>
-       </div>
+          <div class="year">
+            <label for="work_embargo_year" class="visually-hidden">Embargo year</label>
+            <%= select_year Time.zone.today, { prefix: 'work', field_name: 'embargo_date(1i)', start_year: Time.zone.today.year, end_year: 3.years.from_now.year }, id: 'work_embargo_year', class: "form-control" %>
+          </div>
 
-       <div class="day">
-         <label for="work_embargo_day" class="visually-hidden">Embargo day</label>
-         <%= select_day nil, { prefix: 'work', field_name: 'embargo_date(3i)', prompt: 'day'}, id: 'work_embargo_day', class: "form-control" %>
-       </div>
-     </div>
-   </div>
+          <div class="month">
+            <label for="work_embargo_month" class="visually-hidden">Embargo month</label>
+            <%= select_month nil, { prefix: 'work', field_name: 'embargo_date(2i)', prompt: 'month'}, id: 'work_embargo_month', class: "form-control" %>
+          </div>
+
+          <div class="day">
+            <label for="work_embargo_day" class="visually-hidden">Embargo day</label>
+            <%= select_day nil, { prefix: 'work', field_name: 'embargo_date(3i)', prompt: 'day'}, id: 'work_embargo_day', class: "form-control" %>
+          </div>
+        </div>
+      </div>
+      <div data-target="embargo-date.error" class="invalid-feedback"></div>
+      </div>
```

Comment:
Please fix indentation.

Reply:
Should this one be indented? It looks a bit odd as a one liner there, but I think the indentation is correct.

Reply:
shouldn't it be outdented (note there is a close `</div>` on line 36)?